custom-update-agent/go.mod

```diff
@@ -0,0 +1,23 @@
+module github.com/eclipse-kanto/example-applications/custom-update-agent
+
+go 1.18
+
+replace github.com/docker/docker => github.com/moby/moby v23.0.3+incompatible
+
+require (
+	github.com/eclipse-kanto/container-management v0.1.0-M4
+	github.com/eclipse-kanto/update-manager v0.0.0-20230628072101-b91f0c30e00f
```

Comment:
Use the latest update-manager version, as the 0.0.0-20230628072101-b91f0c30e00f points to the initial contribution commit. Afterwards there are some UA API changes that should be reflected in the code.